index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>freeCodeCamp Survey Form</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
Great work in giving this  review. The use of emojis makes it friendly however, for this comment please add an explanation too. This is because, though it might be clear to you, it is not as clear to everyone. This is highlighted in the screenshot below to your requirements.
<img width="1680" alt="Screenshot 2022-09-26 at 12 59 35" src="https://user-images.githubusercontent.com/48546992/192502200-0f4c89af-489f-405a-81e6-a2403fdc760f.png">